### DISCUSSION TITLE
What does precision {0, 1, 2} mean?

### DISCUSSION THREAD

**Discussion by nanassito** [Koenkk/zigbee2mqtt#8371] [General]
There are multiple devices that have a configurable precision. For instance the SNZB-02:

> temperature_precision: Controls the precision of temperature values, e.g. 0, 1 or 2; default 2. To control the precision based on the temperature value set it to e.g. {30: 0, 10: 1}, when temperature >= 30 precision will be 0, when temperature >= 10 precision will be 1. Precision will take into affect with next report of device.

What does it mean for the precision to be `0`, `1`, or `2`? Is it the number of decimal places ?

**Comment by Koenkk** [comment]
> Is it the number of decimal places ?

Yes
